samples/README.md

```diff
@@ -40,6 +40,12 @@ For completeness they are also listed in this directory.
 -   [Methods](methods.go)
 -   [Interfaces](interfaces.go)
 
+### Embedding
+
+-   [Structs](embedding_structs.go)
+-   [Interfaces](embedding_interfaces.go)
+-   [Precedence](embedding_precedence.go)
```

Comment:
These three samples should also be referenced in the slides with links to goplay.space.

Reply:
These were included in a previous pull request, at the time I didn't realise that I also had to include the examples here, so they are being added along with this pull request.